e2e/bzlmod/BUILD

```diff
@@ -4,6 +4,7 @@ load("@bazel_skylib//rules:build_test.bzl", "build_test")
 ts_project(
     name = "ts",
     srcs = ["foo.ts"],
+    supports_workers = False,
```

Comment:
Add a comment / link to issue?

Reply:
What about using bazelrc so that only windows has a disable via the spawn strategy?

Reply:
Chatted about this at the OSS sync. We'll add a feature flag that the rule can check which will disable workers for all ts_project targets in a build. --spawn_stragety=local is too blunt as it applies to everything in the build

Reply:
Actually, simplest is to detect the host platform and disable workers entire on Windows for now. Will do that here.